_topic_map.yml

```diff
@@ -279,6 +279,8 @@ Topics:
         File: manual_upgrades
       - Name: Blue-Green Deployments
         File: blue_green_deployments
+      - Name: Operating System Upgrades
```

Comment:
It's a bit of a placeholder for reference from a kbase article and release notes for the OS.  But it would be nice if it were more prominent in the docs too. @adellape can you suggest somewhere better.